library.js

```diff
@@ -144,24 +143,21 @@ plugin.getUser = async (remoteId) => {
 
 plugin.process = async (token, request, response) => {
   try {
-    const userInfo = await validateToken(token, userInfoUrl);
+    const { isValidMember, userInfo } = await getFeideUser(
+      token,
+      feiderUserUrl,
+      validRoles,
+    );
     if (!userInfo) {
       response.status(403).send('Forbidden');
       return;
     }
 
-    const { isValidMember, uid } = await validateMemberStatus(
-      token,
-      memberStatusUrl,
-      validRoles,
-    );
-
     if (!isValidMember) {
       response.status(403).send('Forbidden');
       return;
     }
-
-    const userDataResult = { ...userInfo, uid };
+    const userDataResult = { ...userInfo };
     const userData = userDataResult ? userDataResult : null;
```

Comment:
Vil ikkje dette gi det samme? Og er det litt rart å sette null på userdata?
```suggestion
    const userData = userInfo ? userInfo : null;
```